src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -281,12 +282,43 @@ public sealed class ProgressConfiguration
             /// <summary>
             /// Gets or sets the style for progress bar.
             /// </summary>
-            public string Style { get; set; } = "\x1b[33;1m";
+            public string Style
+            {
+                get
+                {
+                    return _style;
+                } 
+
+                set
+                {
+                    _style = ValidateNoContent(value);
+                }
+            }
```

Comment:
It would be nice to use modern syntax:
```suggestion
            public string Style
            {
                get => _style;
                set => _style = ValidateNoContent(value);
            }
```

Reply:
will update